integration_testing/features/super-admin/ORDER.txt

```diff
@@ -1,35 +1,53 @@
-super-admin-windows-installer-translated-strings.feature
-super-admin-windows-installer-setup-error-log.feature
-super-admin-setup-wizard.feature
+super-admin-quick-start-setup-wizard.feature
+super-admin-creates-new-facility-in-device-setup.feature
+super-admin-imports-facility-in-device-setup.feature
 super-admin-sign-in.feature
+super-admin-sign-out.feature
+device/super-admin-select-import-content-location.feature
```

Comment:
@mrpau-dev 
The idea was to make the folder with super admin Gherkins more manageable, as we have breached 50, and grouping them by plugin (`device`, `facility`...) or feature (Terminal commands and those that involve playing with `options.ini`), might alleviate that.

What I didn't think of is that you will need to amend the script that generates the spreadsheet... :facepalm: 
Please let me know if this version of makes that doable without much effort!

Note that some feature files are _general_ and not in the sub-folders, but in order to have an approximate (recommended) order of running through the scenarios, they are a mix (few from one folder, then few from the other, then few from the root folder). Let me know if this makes sense, thank you!

Reply:
@radinamatic I think we need to create a list of those general feature files and where it was applicable and put them into text file then let the script distribute them. maybe add the group name of the feature in to first name of the feature sample `device-facility-common-multiple-facility-sign-in.feature ` that feature file is applicable at `device` and `facility` groups

Reply:
The idea is to have the `feature` files on the repo in smaller groups, to make future reviews and updates easier, but the spreadsheet should look the same way as it is now (all super-admin scenarios in the same group of rows, as the other roles). Since the spreadsheet has a tabular structure and we determine the order, it's easy to follow, right?
Changing the names of the `feature` files to include the _denomination_ of the group would not change anything... :confused:

Reply:
> Since the spreadsheet has a tabular structure and we determine the order, it's easy to follow, right?

Yes @radinamatic we just need to update the `ORDER.txt` to determine the ordering of the feature files at the spreadsheet.

Reply:
That's what I tried to do in the `ORDER.txt` file in the super-admin folder: I reviewed the order and indicated all the files according to their new position - if they are in a sub-folder, it is stated before the feature file name? Is that not correct?

Reply:
I think that's just fine, I just have a minor changes in the script to make that work.

Reply:
@mrpau-richard There's an [existing PR for the changes I requested here](https://github.com/radinamatic/kolibri/pull/5) - maybe you can use that one?